frontend/src/components/agency/logs-label/LogsLabel.jsx

```diff
@@ -1,37 +1,50 @@
 import { Col, Row, Typography } from "antd";
 
 import "./LogsLabel.css";
+import { useLocation } from "react-router-dom";
 
 function LogsLabel() {
+  const location = useLocation(); // Get the current route location
+  const isWorkflowSubPage = /^\/[^/]+\/workflows\/.+/.test(location.pathname);
+  const isPromptStudioPage = /^\/[^/]+\/tools\/.+/.test(location.pathname);
```

Comment:
```
const isSubPage(type) => {
  const regex = new RegExp(`^\/[^/]+\/${type}\/.+`);
  return regex.test(location.pathname);
};
```

Can we add a function like this and use it here? The function can be placed in `unstract/frontend/src/helpers/GetStaticData.js.`